cobigen/cobigen-javaplugin-parent/cobigen-javaplugin/src/main/java/com/devonfw/cobigen/javaplugin/merger/libextension/ModifyableModelBuilder.java

```diff
@@ -185,17 +210,27 @@ public void beginClass(ClassDef def) {
     }
 
     protected ModifyableJavaClass bindClass(ModifyableJavaClass newClass) {
-
         if (currentField != null) {
             classStack.getFirst().addClass(newClass);
             currentField.setEnumConstantClass(newClass);
         } else if (!classStack.isEmpty()) {
             classStack.getFirst().addClass(newClass);
-            newClass.setParentClass(classStack.getFirst());
+            newClass.setDeclaringClass(classStack.getFirst());
```

Comment:
did you compare and change such changes based on the DefaultModelBuilder?

Reply:
setDeclaringClass seems to simply be the renamed setParentClass.
In general, the term "parent" seems to have been replaced by "declaring".